qmlti/lti/LTI_Tool_Provider.php

```diff
@@ -209,11 +208,16 @@ public function execute() {
     if (isset($_POST['launch_presentation_return_url'])) {
       $this->return_url = $_POST['launch_presentation_return_url'];
     }
+
 #
 ### Perform action
 #
     if ($this->authenticate()) {
       $this->doCallback();
+      error_log("Callback successful.");
```

Comment:
Is this really just error logging? you seem to be using it for info messages as well...